velox/common/memory/SharedArbitrator.cpp

```diff
@@ -345,10 +358,25 @@ uint64_t SharedArbitrator::testingNumRequests() const {
   return numRequests_;
 }
 
+uint64_t SharedArbitrator::adjustedGrowCapacity(
+    const MemoryPool& pool,
+    uint64_t targetBytes) const {
+  uint64_t adjustedTargetBytes{0};
+  const auto capacity = pool.capacity();
+  if (capacity * 2 < 512ULL << 20) {
+    adjustedTargetBytes = capacity;
```

Comment:
Can we round up to the next power of 2 to ease debugging?

Reply:
Why is it going to be easing debugging?